src/Collection/DefaultIterator.php

```diff
@@ -361,6 +362,14 @@ public function setSortFunction(callable $sortFunction)
         return $this;
     }
 
+    /**
+     * @return callable
+     */
+    public function getSortFunction(): callable
```

Comment:
Unless this is imposed by the parent interface, I would drop it. Checking via reflection is still preferable to adding public API

Reply:
Done in PR #13 - I will update here.